### PR TITLE
Genie merge

### PR DESCRIPTION
**Pull request by bwalsh**
This PR has no new features.  
It has maintenance items:
* moves genie analysis to a `notebooks/` folder
* adds README.md to document synapse dependency
* adds bokeh dependency

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# notebook 
+
+## external dependencies
+  https://www.synapse.org/#!Synapse:syn7851250 copy to data_mutations_extended_1.0.1.txt
```
Comment (bwalsh) [inline notebooks/README.md:4]: //TODO - create makefile or some other dependency mgt solution see #65

**Comment by bwalsh** [comment]
@jgoecks  - can you review?

**Comment by jgoecks** [comment]
Looks good, +1 from me.

**Comment by bwalsh** [comment]
merged

**Pull request closed without merging**